src/appendices/string-method-examples/replace-examples.rst

```diff
@@ -0,0 +1,46 @@
+.. _string-replace-examples:
+
+**replace** Examples
+====================
+
+The general syntax for this method is:
+
+.. sourcecode:: js
+
+   stringName.replace(searchChar, replacementChar);
+
+Given a search string ``searchChar`` and a replacement value ``replacementChar``, this method returns a copy of ``stringName`` with the *first* occurence of ``searchChar`` replaced by ``replacementChar``.
```

Comment:
Same suggestion from the string methods page.  How about ``newChar`` instead of ``replacementChar``?